src/ralph/api/utils.py

```diff
@@ -69,7 +73,9 @@ def get_serializer(self, *args, **kwargs):
                     args[0].__class__
                 ]
             except KeyError:
-                pass
```

Comment:
imo yes. If something has no serializer we don't know how to serialize it (on dedicated view), so only base object serializer (or any other polymorphic in the future) will be used and only fields of base object will be returned.